server/WebAPI/Services/ScheduledPlaceToEatService.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using Entities.Interfaces;
+using Entities.Models;
+using WebAPI.DTO.ScheduledPlaceToEat;
+using WebAPI.Interfaces;
+
+namespace WebAPI.Services
+{
+  public class ScheduledPlaceToEatService : IScheduledPlaceToEatService
+  {
+    private readonly UserService _userService;
+    private readonly IScheduledPlaceToEatRepository _scheduledPlaceToEatRepository;
+    private readonly IAttachmentFileEatingRepository _attachmentFileEatingRepository;
+
+    public ScheduledPlaceToEatService(
+      UserService userService,
+      IScheduledPlaceToEatRepository scheduledPlaceToEatRepository,
+      IAttachmentFileEatingRepository attachmentFileEatingRepository)
+    {
+      _userService = userService;
+      _scheduledPlaceToEatRepository = scheduledPlaceToEatRepository;
+      _attachmentFileEatingRepository = attachmentFileEatingRepository;
+    }
+
+    public int CreateNewEating(InputScheduledPlaceToEatForCreateDTO scheduledPlaceToEatDTO)
+    {
+      try
+      {
+        if(!Valid(scheduledPlaceToEatDTO))
+        {
+          throw new ArgumentException();
+        }
+        else
+        {
+          if(_userService.GetUser(scheduledPlaceToEatDTO.UserId) != null)
+          {
+            return _scheduledPlaceToEatRepository.CreateScheduledPlaceToEat(ConvertInputScheduledPlaceToEatForCreateDTOToScheduletPlaceToEat(scheduledPlaceToEatDTO));
+          }
+          else
+          {
+            throw new ArgumentException();
+          }
+        }
+      }
+      catch(Exception)
+      {
+        return -1;
```

Comment:
what does this -1 mean?

Reply:
-1 indicates that the creation was not successful and the controller will analyze this and return an error